public/javascripts/SVValidate/src/status/StatusExample.js

```diff
@@ -57,12 +57,15 @@ function StatusExample (statusUI) {
     }
 
     /**
-     * Sets the horizontal position and height of the popup based on which picture was hovered over.
+     * Sets the horizontal position, vertical position, and height of the popup
+     * as well as the popup pointer basedon which picture was hovered over.
      * @param id    ID name for the label example HTML element that the user hovered over.
      * @private
      */
     function _setPopupLocation (id) {
         // 1 = upper left, 2 = upper right, 3 = bottom left, 4 = bottom right
+
+        // Horizontal positioning
```

Comment:
Can you add a period to the end of this comment and the following comment, to adhere to the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-Guide)?

Reply:
Even though the rest of the comments don't have periods at the end?

Reply:
Yeah, we're slowly converting everything over to this method :sweat_smile: if you're feeling ambitious, you can feel free to edit the other comments to fit the style too :grin: